build/windows/installer/Package.wxs

```diff
@@ -0,0 +1,51 @@
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:ui="http://wixtoolset.org/schemas/v4/wxs/ui">
+    <!-- ToDo use your own keys for Name and Manufacturer, as well as a new UUID for UpgradeCode -->
+    <Package
+        Name="BevyGame"
+        Manufacturer="My Name"
+        UpgradeCode="132bb59d-de4f-4893-a535-93dfd0597731"
+        Version="0.1.0"
+
```

Comment:
Could this line break be removed?

Reply:
It could, but I advise against it since it cleanly separates the part the user should adapt to their use-case from the stuff the user doesn't need to touch

Reply:
Ok, thank you for considering this! ✨